lib/templates/html/_sidebar_for_container.html

```diff
@@ -64,7 +66,9 @@
     <li>{{{ linkedName }}}</li>
     {{/publicInstanceOperatorsSorted}}
     {{/hasPublicInstanceOperators}}
+    {{/isExtension}}
 
+    {{#isClassOrExtension}}
```

Comment:
mustache may not require it but closing this section seems like a good idea

Reply:
I think this one is closed later on in the file, it just isn't included in the changes as it was pre-existing.

Reply:
oh, you're right I missed that.